notebooks/Project and Developer Similarity.ipynb

```diff
@@ -0,0 +1,625 @@
+{
```

Comment:
<p>Did you consider explaining a general concept of sparsity a bit more in here?</p><p> </p><p>Also, may be a bit more clear highlight of what have changed before/after in order to add sparsity? Asking, as from a quick glance I could not tell if that's only a regulation parameter or is there anything else.</p><p> </p><p>One way to make it more obvious may be for the <code>sparsify_topic_model</code> to delegate to <code>train_topic_model()</code> by calling it with override keyword arguments. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mloncode/devfest2019-workshop/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Thanks for pointing that out! I will clarify with comments in a following PR :)</p><p> </p><p>As for reusing <code>train_topic_model</code>, I'm not sure it will be clearer: in one case we set the number of topics, in the other we reuse an existing model, it will make logic (a tad) more complex. Let me know if you think it'd be better though, I'm not against it!</p><p> </p>